snmp/datadog_checks/snmp/snmp.py

```diff
@@ -56,10 +57,22 @@ def __init__(self, name, init_config, instances):
         # Load Custom MIB directory
         self.mibs_path = init_config.get('mibs_folder')
         self.ignore_nonincreasing_oid = is_affirmative(init_config.get('ignore_nonincreasing_oid', False))
+        self.profiles = init_config.get('profiles', {})
+        for profile, profile_data in list(self.profiles.items()):
```

Comment:
```suggestion
        for profile, profile_data in self.profiles.items():
```

Reply:
Yeah I usually prefer to copy the dict when I change it, but in this case I modify the entries in place so it shouldn't matter.